coordinator/src/main/scala/filodb.coordinator/QueryActor.scala

```diff
@@ -203,11 +236,11 @@ final class QueryActor(memStore: TimeSeriesStore,
       t match {
         case _: BadQueryException => // dont log user errors
         case _: AskTimeoutException => // dont log ask timeouts. useless - let it simply flow up
-        case e: QueryTimeoutException => // log just message, no need for stacktrace
-          logger.error(s"queryId: ${q.queryContext.queryId} QueryTimeoutException: " +
-            s"${q.queryContext.origQueryParams} ${e.getMessage}")
+        case _: QueryTimeoutException | _: ExecutionRejectedException => // log just message, no need for stacktrace
+          logger.error(s"Query Error ${t.getClass.getSimpleName} queryId=${q.queryContext.queryId} " +
+            s"${q.queryContext.origQueryParams} ${t.getMessage}")
```

Comment:
This is  ``ExecutionRejectedException``

```
class ExecutionRejectedException(val message: String, cause: Throwable)
  extends RuntimeException(message, cause) with Serializable {

  def this(message: String) = this(message, null)
  def this(cause: Throwable) = this(null, cause)
}
```

If it is constructed using just root cause, we will miss the exception in logs as the message null and root cause is not logged. Im not sure where and how it might be instantiated this way.

Reply:
ExecutionRejectedException typically does not have an inner cause. This is what I have seen. Will skip for now - can add later if needed.